history.js

```diff
@@ -413,7 +422,7 @@
    */
   function parseURL(href, isWindowLocation, isNotAPI) {
     var re = /(?:([a-zA-Z0-9\-]+\:))?(?:\/\/(?:[^@]*@)?([^\/:\?#]+)(?::([0-9]+))?)?([^\?#]*)(?:(\?[^#]+)|\?)?(?:(#.*))?/;
-    if (href !== null && href !== '' && !isWindowLocation) {
+    if (isPresent(href) && href !== '' && !isWindowLocation) {
```

Comment:
Yes, you are right. I include that because think is more cleaner when you want to check if a value is undefined and not null, and think can help to prevent this type of errors with a minimum of overhead.

Greatings!